src/test/scala/com/microsoft/ml/spark/cognitive/split1/TextAnalyticsSDKSuite.scala

```diff
@@ -134,17 +210,29 @@ class KeyPhraseExtractionSuiteV4 extends TestBase with DataFrameEquality with Te
     assert(replies(0).getSeq[String](0).toSet == Set("Hello world", "input text"))
   }
 
-    test("KPE - Output Assertion"){
-      val replies = extractor.transform(df2)
-        .select("output")
-        .collect()
+  test("KPE - Output Assertion") {
+    val replies = extractor.transform(df2)
+      .select("output")
+      .collect()
 
-      assert(replies(0).schema(0).name == "output")
+    assert(replies(0).schema(0).name == "output")
 
-      df2.printSchema()
-      df2.show()
-      replies.foreach { row =>
-        row.toSeq.foreach { col => println(col) }
-      }
+    df2.printSchema()
+    df2.show()
+    replies.foreach { row =>
+      row.toSeq.foreach { col => println(col) }
     }
+  }
+  test("KPE - Check Model Version") {
+    val replies = extractor.transform(df2)
+      .select("output")
+      .collect()
+    assert(replies(0).schema(0).name == "output")
+    val fromRow = KeyPhraseResponseV4.makeFromRowConverter
+    replies.foreach(row => {
+      val outResponse = fromRow(row.getAs[GenericRowWithSchema]("output"))
+      val modelCheck = outResponse.result.head.get
+      modelCheck.toString.matches("\\d{4}-\\d{2}-\\d{2}")
+    })
+  }

```

Comment:
Also, please resolve the conflicts between the current branch with the internship-ta branch.